src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/NotificationServiceTest.java

```diff
@@ -104,7 +105,7 @@ void should_not_send_notification_when_notification_was_already_sent() {
         // given
         // rejected and notification_sent
         var envelopeId1 = envelopeService.createNewEnvelope("bulkscan", "blob1.zip", now());
-        envelopeService.markAsRejected(envelopeId1, "duplicate file");
+        envelopeService.markAsRejected(envelopeId1, ErrorCode.ERR_METAFILE_INVALID, "duplicate file");
```

Comment:
blob router does not know about metafile, a different code should be used

Reply:
update to use `ERR_ZIP_PROCESSING_FAILED` error code